opentelemetry-sdk/src/metrics/internal/sum.rs

```diff
@@ -25,40 +25,54 @@ struct ValueMap<T: Number<T>> {
     has_no_value_attribute_value: AtomicBool,
     no_attribute_value: T::AtomicTracker,
     count: AtomicUsize,
+    assign_only: bool, // if true, only assign incoming value, instead of adding to existing value
```

Comment:
Could we rename this to something more intuitive like `is_sum_precomputed`? I think that would better explain the reasoning behind the logic when reading the code:

```rust
if self.is_sum_precomputed {
    value_to_update.store(measurement);
} else {
    value_to_update.add(measurement);
}
```